File: app/src/main/java/com/datarockets/mnchkn/activities/players/PlayersListPresenter.java
package com.datarockets.mnchkn.activities.players;

public interface PlayersListPresenter {
    void checkIsEnoughPlayers();
    void addPlayer(String name);
    void deletePlayerListItem(int position, long id);
    void clearPlayersStats();
    void clearGameSteps();
    void setGameStarted();
    void setGameFinished();
    void onCreate();
    void onResume();
    void onDestroy();
}

File: app/src/main/kotlin/com/datarockets/mnchkn/activities/players/PlayersListPresenter.kt
package com.datarockets.mnchkn.activities.players

interface PlayersListPresenter {
    fun checkIsEnoughPlayers()
    fun addPlayer(name: String)
    fun deletePlayerListItem(position: Int, id: Long)
    fun clearPlayersStats()
    fun clearGameSteps()
    fun setGameStarted()
    fun setGameFinished()
    fun onCreate()
    fun onResume()
    fun onDestroy()
}
